pkg/apis/routing/v1alpha1/filter_types.go

```diff
@@ -78,7 +78,7 @@ func (f *Filter) GetStatus() *duckv1.Status {
 	return &f.Status.Status
 }
 
-// AsRouter implements Router.
-func (f *Filter) AsRouter() string {
+// AsEventSource implements Router.
+func (f *Filter) AsEventSource() string {
 	return "filter/" + f.Name
 }
```

Comment:
I didn't touch the logic here, but it's not particularly helpful because we don't know the namespace of the object.
Mind if I add it here too?